ecosystem/node-checker/src/evaluators/direct/api/transaction_presence.rs

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{super::DirectEvaluatorInput, ApiEvaluatorError, API_CATEGORY};
+use crate::{
+    configuration::EvaluatorArgs,
+    evaluator::{EvaluationResult, Evaluator},
+    evaluators::EvaluatorType,
+};
+use anyhow::{anyhow, Result};
+use aptos_rest_client::{aptos_api_types::TransactionInfo, Client as AptosRestClient, Transaction};
+use aptos_sdk::crypto::HashValue;
+use clap::Parser;
+use poem_openapi::Object as PoemObject;
+use serde::{Deserialize, Serialize};
+use std::time::Duration;
+
+#[derive(Clone, Debug, Deserialize, Parser, PoemObject, Serialize)]
+pub struct TransactionPresenceEvaluatorArgs {
+    #[clap(long, default_value_t = 5)]
+    pub transaction_fetch_delay_secs: u64,
+}
+
+#[allow(dead_code)]
```

Comment:
Why do we need this? 🤔

Reply:
Copy pasta, not necessary here but in other evaluators where args is empty, we need it to silence the linter. I prefer to pipe arguments through just in case we want to add them later to an evaluator.